### PR TITLE
some cleanup and translation

### PR DESCRIPTION
**Pull request by fabianfrz**

### CODE REVIEW

File: src/www/interfaces.php [hunk 1]
```diff
@@ -2052,7 +2052,7 @@ function toggle_allcfg() {
                               <?=gettext("The value in this field is the full absolute path to a DHCP client configuration file.  [/[dirname/[.../]]filename[.ext]] <br /> " .
                               "Value Substitutions in Config File: {interface}, {hostname}, {mac_addr_asciiCD}, {mac_addr_hexCD} <br />" .
                               "Where C is U(pper) or L(ower) Case, and D is \" :-.\" Delimiter (space, colon, hyphen, or period) (omitted for none). <br />" .
```
Comment (fabianfrz) [inline src/www/interfaces.php]: @fichtner includes HTML-Tags

File: src/www/diag_ipsec_leases.php [hunk 2]
```diff
@@ -55,17 +55,17 @@
                       <tr>
                         <th></th>
                         <th colspan="3">
-                          <?=gettext("Pool: ");?> <?=$pool;?>
-                          <?=gettext("usage: ");?> <?=$pool_data['usage'];?>
-                          <?=gettext("online: ");?> <?=$pool_data['online'];?>
+                          <?= gettext("Pool:") ?> <?= $pool ?>
+                          <?= gettext("usage:") ?> <?= $pool_data['usage'] ?>
+                          <?= gettext("online:") ?> <?= $pool_data['online'] ?>
```
Comment (fichtner) [inline src/www/diag_ipsec_leases.php]: these two should be uppercase

File: src/www/interfaces.php (hunk 1 above)
Comment (fichtner) [inline src/www/interfaces.php]: let's leave this as is for now

File: src/www/interfaces.php [hunk 3]
```diff
@@ -1888,7 +1888,7 @@ function toggle_allcfg() {
                             <div class="hidden" for="help_for_gateway">
                               <?=gettext("If this interface is an Internet connection, select an existing Gateway from the list or add a new one using the link above."); ?><br />
                               <?=gettext("On local LANs the upstream gateway should be \"none\"."); ?>
-                              <p><strong><?=gettext("NOTE:"); ?></strong> <?=gettext("You can manage Gateways"); ?> <a target='_blank' href='system_gateways.php'><?=gettext("here"); ?></a>.</p>
+                              <p><strong><?=gettext("NOTE:"); ?></strong> <?= sprintf(gettext("You can manage Gateways %shere%s."),'<a target="_blank" href="system_gateways.php">','</a>') ?></p>
```
Comment (fichtner) [inline src/www/interfaces.php]: kill strong + NOTE, or the whole line if you want.

File: src/www/interfaces.php [hunk 4]
```diff
@@ -2359,11 +2359,11 @@ function toggle_allcfg() {
...
+                            <button type="button" class="btn btn-sm" id="btn_show_add_gatewayv6" title="<?=gettext("Add a new one.");?>"  data-toggle="tooltip"><span class="glyphicon glyphicon-plus"></span></button>
                             <div class="hidden" for="help_for_gatewayv6">
                               <?=gettext("If this interface is an Internet connection, select an existing Gateway from the list or add a new one using the link above."); ?><br />
-                              <?=gettext("On local LANs the upstream gateway should be \"none\"."); ?>
-                              <p><strong><?=gettext("NOTE:"); ?></strong> <?=gettext("You can manage Gateways"); ?> <a target='_blank' href='system_gateways.php'><?=gettext("here"); ?></a>.</p>
+                              <?=gettext('On local LANs the upstream gateway should be "none".'); ?>
+                              <p><strong><?=gettext("NOTE:"); ?></strong> <?= sprintf(gettext("You can manage Gateways %shere%s."),'<a target="_blank" href="system_gateways.php">','</a>') ?></p>
```
Comment (fichtner) [inline src/www/interfaces.php]: zap strong+NOTE

File: src/www/services_dhcp.php [hunk 5]
```diff
@@ -708,7 +708,7 @@ function removeRow() {
                       <td><i class="fa fa-info-circle text-muted"></i> <?=gettext("Enable");?> </td>
                       <td>
                         <input name="enable" id="enable" type="checkbox" value="yes" <?=!empty($pconfig['enable']) ? "checked=\"checked\"" : ""; ?> />
-                        <strong><?php printf(gettext("Enable DHCP server on " . "%s " ."interface"),!empty($config['interfaces'][$if]['descr']) ? htmlspecialchars($config['interfaces'][$if]['descr']) : strtoupper($if));?></strong>
+                        <strong><?php printf(gettext("Enable DHCP server on %s interface"),!empty($config['interfaces'][$if]['descr']) ? htmlspecialchars($config['interfaces'][$if]['descr']) : strtoupper($if));?></strong>
```
Comment (fichtner) [inline src/www/services_dhcp.php]: on *the* %s interface

**Comment by fichtner** [comment]
Thanks! :)

**Pull request merged**